src/lib/LUA/rx_devLUA.cpp

```diff
@@ -785,16 +785,16 @@ static void luaparamMappingChannelOut(struct luaPropertiesCommon *item, uint8_t
             pModeString = serial1_TX;
         }
         else
-        { 
+        {
```

Comment:
I don't know why this happened. I don't have any custom git eol settings, there is just the text=auto in gitattributes. maybe this was an issue in the original file and neovim or git changed it?